src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -18,6 +18,7 @@ internal static partial class SocketPal
         public const bool SupportsMultipleConnectAttempts = false;
         public static readonly int MaximumAddressSize = Interop.Sys.GetMaximumAddressSize();
         private static readonly bool SupportsDualModeIPv4PacketInfo = GetPlatformSupportsDualModeIPv4PacketInfo();
+        private const int IovStackThreshold = 8;
```

Comment:
This is a very safe value, it could be set higher. It matches the same value used by Linux kernel for stackallocing iovs.

Reply:
Can you add a comment as to how the value was chosen?